pkg/inspector/image-inspector.go

```diff
@@ -499,27 +526,28 @@ func (i *defaultImageInspector) getAuthConfigs() (*docker.AuthConfigurations, er
 	return imagePullAuths, nil
 }
 
-func (i *defaultImageInspector) scanImage(s iiapi.Scanner) ([]byte, []byte, error) {
+// TODO: Move this into openscap package
```

Comment:
ok, then maybe we don't need this function here or to change the interface. This is just meant to reduce duplication from handling multiple scanners but as it was written with only one scanner it is obviously not sufficient.